### ISSUE TITLE
WebView catches focus automatically after loading

### ISSUE THREAD

**Issue by Nutznieser** [wxWidgets/Phoenix#414]
Following situation: I have a wx.ListCtrl and a webview.WebView in one window together. Now I want to update some html code within the webview (it acts as an information view) if I click on a list item in the listctrl. But after the new html code is loaded within the webview, it catches the focus automatically. The webview also loads the html code asynchronous. So a listctrl.SetFocus() call directly after executing the loading will have no effect.

Is there a way to prevent that the webview will catching the focus automatically?

**Comment by vScourge** [comment]
I'm also seeing this, and would be curious if a workaround has been found?  I'm currently using 4.0.0a3

**Comment by qalexma** [comment]
@Nutznieser did you find any solution by any chance? Please let me know, I'm having the same issue

**Comment by Nutznieser** [comment]
@qalexma Unfortunately I couldn't found any solution for this. I tested it also with the newest wxPython version 4.1 but there is the same effect.

Currently I use the integrated generic module [wx.html.HtmlWindow](https://wxpython.org/Phoenix/docs/html/wx.html.HtmlWindow.html?highlight=wx%20html%20htmlwindow#wx.html.HtmlWindow) instead. Until now it is sufficient for my functionality.

**Comment by qalexma** [comment]
Fair enough, will try that one (not sure if it will be able to handle it though, too much css in my content). Thanks for the answer, much appreciated!
